### DISCUSSION TITLE
docker-compose 작성 시 컨테이너간 url

### DISCUSSION THREAD

**Discussion by sgb8170** [Gyubam/bf-matching#1] [Q&A] [answered]
url 주소의 경우 컨테이너의 퍼블릭 ip 주소로 설정해야 하나요?

**Comment by Gyubam** [comment, marked as answer]
![132521352135](https://user-images.githubusercontent.com/71515740/193441530-161d7f85-e712-4ab2-b2c6-7a04f164e730.PNG)


![123123123124124](https://user-images.githubusercontent.com/71515740/193441534-63cd3b76-ba91-4d29-b262-8bd0f15e4001.PNG)

docker-compose를 통해 네트워크 주소로 설정할 경우 서비스 명을 사용할 수 있습니다
